flowapi/flowapi/api_spec.py

```diff
@@ -53,13 +53,21 @@ async def get_spec(socket: Socket, request_id: str) -> APISpec:
         ),
     )
     spec.components.schemas.update(flowmachine_query_schemas)
+    scopes = schema_to_scopes(spec.to_dict())
+    scopes = [
```

Comment:
I know this bit is my code, but can we inline this to avoid multiply creating the massive list?